src/client/DataStandardManager.ts

```diff
@@ -103,3 +103,21 @@ export async function ensureBundledDsReadOnly() {
     }
   }
 }
+
+/**
+ * Returns value indicating whether data standard project is in workspace
+ */
+export function isDataStandardProjectInWorkspace(): boolean {
```

Comment:
Please add "bundled" to function name. (DS team use case doesn't use bundled DSes.)